app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java

```diff
@@ -122,5 +126,6 @@ public String getText() {
 
     public void setText(String text) {
```

Comment:
I wonder what the use of this function! :thinking: in the onclick implementation it's the same lines is it?

Reply:
Actually the onClick implementation is for the button clicks. When the value gets incremented or decremented the cursor will also remain at the end.
While this setText method is for EditTextWidget so when this dialog is opened,the cursor will be seen at the end